<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>卡拉彼丘爆破2.0枪械表现计算器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QGridLayout" name="layout_main" rowstretch="0,1,0" columnstretch="0,1,0">
     <item row="0" column="1">
      <spacer name="vs_up">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <spacer name="vs_down">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <spacer name="hs_right">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="hs_left">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="widget_main" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>选择版本</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>选择数据版本:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="data_version_box"/>
           </item>
           <item>
            <widget class="QPushButton" name="load_button">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>加载</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>10m无加点上身200血数据</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableWidget" name="tab1_table">
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
              <property name="dragEnabled">
               <bool>false</bool>
              </property>
              <property name="dragDropOverwriteMode">
               <bool>false</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::DragDropMode::NoDragDrop</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::DropAction::IgnoreAction</enum>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
              </property>
              <column>
               <property name="text">
                <string>超弦体</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>射速</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>伤害</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>DPS</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>TTK</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>计算器</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,1">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,0,0">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QGroupBox" name="groupBox_2">
                     <property name="title">
                      <string>超弦体</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_4">
                      <item row="0" column="0">
                       <widget class="QComboBox" name="tab2_agents_box"/>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGroupBox" name="groupBox_3">
                     <property name="title">
                      <string>基础伤害加成</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_3">
                      <item row="0" column="0">
                       <widget class="QComboBox" name="tab2_damage_box">
                        <item>
                         <property name="text">
                          <string>lv0</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv1</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv2</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv3</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv4</string>
                         </property>
                        </item>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <widget class="QGroupBox" name="groupBox_4">
                     <property name="title">
                      <string>基础射速加成</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_2">
                      <item row="0" column="0">
                       <widget class="QComboBox" name="tab2_rof_box">
                        <item>
                         <property name="text">
                          <string>lv0</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv1</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv2</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv3</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv4</string>
                         </property>
                        </item>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGroupBox" name="groupBox_5">
                     <property name="title">
                      <string>弹丸数提升</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_5">
                      <item row="0" column="0">
                       <widget class="QComboBox" name="tab2_bullet_box">
                        <item>
                         <property name="text">
                          <string>lv0</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv1</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv2</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv3</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv4</string>
                         </property>
                        </item>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <widget class="QGroupBox" name="groupBox_6">
                     <property name="title">
                      <string>爆头伤害提升</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_6">
                      <item row="1" column="0">
                       <widget class="QComboBox" name="tab2_head_box">
                        <item>
                         <property name="text">
                          <string>lv0</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv1</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv2</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv3</string>
                         </property>
                        </item>
                        <item>
                         <property name="text">
                          <string>lv4</string>
                         </property>
                        </item>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QGroupBox" name="groupBox_7">
                     <property name="title">
                      <string>目标血量</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_7">
                      <item row="0" column="0">
                       <widget class="QSpinBox" name="spinBox">
                        <property name="enabled">
                         <bool>false</bool>
                        </property>
                        <property name="frame">
                         <bool>true</bool>
                        </property>
                        <property name="buttonSymbols">
                         <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
                        </property>
                        <property name="keyboardTracking">
                         <bool>true</bool>
                        </property>
                        <property name="maximum">
                         <number>1000</number>
                        </property>
                        <property name="value">
                         <number>200</number>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="choose_tableWidget">
                   <property name="layoutDirection">
                    <enum>Qt::LayoutDirection::LeftToRight</enum>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Shape::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Shadow::Sunken</enum>
                   </property>
                   <property name="autoScroll">
                    <bool>false</bool>
                   </property>
                   <property name="editTriggers">
                    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                   </property>
                   <property name="tabKeyNavigation">
                    <bool>false</bool>
                   </property>
                   <property name="showDropIndicator" stdset="0">
                    <bool>false</bool>
                   </property>
                   <property name="dragDropOverwriteMode">
                    <bool>false</bool>
                   </property>
                   <property name="selectionMode">
                    <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
                   </property>
                   <property name="textElideMode">
                    <enum>Qt::TextElideMode::ElideMiddle</enum>
                   </property>
                   <property name="verticalScrollMode">
                    <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
                   </property>
                   <property name="horizontalScrollMode">
                    <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
                   </property>
                   <property name="gridStyle">
                    <enum>Qt::PenStyle::SolidLine</enum>
                   </property>
                   <attribute name="horizontalHeaderVisible">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="horizontalHeaderCascadingSectionResizes">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="horizontalHeaderMinimumSectionSize">
                    <number>40</number>
                   </attribute>
                   <attribute name="horizontalHeaderDefaultSectionSize">
                    <number>60</number>
                   </attribute>
                   <attribute name="horizontalHeaderHighlightSections">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="verticalHeaderVisible">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="verticalHeaderCascadingSectionResizes">
                    <bool>true</bool>
                   </attribute>
                   <attribute name="verticalHeaderMinimumSectionSize">
                    <number>20</number>
                   </attribute>
                   <attribute name="verticalHeaderDefaultSectionSize">
                    <number>50</number>
                   </attribute>
                   <attribute name="verticalHeaderHighlightSections">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                    <bool>false</bool>
                   </attribute>
                   <row>
                    <property name="text">
                     <string>头</string>
                    </property>
                   </row>
                   <row>
                    <property name="text">
                     <string>身</string>
                    </property>
                   </row>
                   <row>
                    <property name="text">
                     <string>脚</string>
                    </property>
                   </row>
                   <column>
                    <property name="text">
                     <string>10m</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>30m</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>50m</string>
                    </property>
                   </column>
                   <item row="0" column="0">
                    <property name="text">
                     <string>9</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                   <item row="0" column="1">
                    <property name="text">
                     <string>8</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                   <item row="0" column="2">
                    <property name="text">
                     <string>7</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                   <item row="1" column="0">
                    <property name="text">
                     <string>6</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                   <item row="1" column="1">
                    <property name="text">
                     <string>5</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                   <item row="1" column="2">
                    <property name="text">
                     <string>4</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                   <item row="2" column="0">
                    <property name="text">
                     <string>3</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                   <item row="2" column="1">
                    <property name="text">
                     <string>2</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                   <item row="2" column="2">
                    <property name="text">
                     <string>1</string>
                    </property>
                    <property name="textAlignment">
                     <set>AlignCenter</set>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <widget class="QPushButton" name="calc_button">
                     <property name="text">
                      <string>计算</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton">
                     <property name="text">
                      <string>清空</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTableWidget" name="tab2_table">
                 <property name="editTriggers">
                  <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
                 </property>
                 <property name="dragDropOverwriteMode">
                  <bool>false</bool>
                 </property>
                 <property name="verticalScrollMode">
                  <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
                 </property>
                 <property name="horizontalScrollMode">
                  <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
                 </property>
                 <column>
                  <property name="text">
                   <string>超弦体</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>射速</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>伤害</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>DPS</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>TTK</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>DPS算是爆破2.0相当不重要的数据了</string>
            </property>
            <property name="textFormat">
             <enum>Qt::TextFormat::MarkdownText</enum>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Github([UI](https://github.com)) ([Data](https://github.com))</string>
            </property>
            <property name="textFormat">
             <enum>Qt::TextFormat::MarkdownText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
